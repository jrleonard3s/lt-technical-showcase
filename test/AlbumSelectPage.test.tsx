import { describe, expect, it } from "vitest";
import "@testing-library/jest-dom/vitest";

describe("AlbumSelectPage", () => {
  it("placeholder", () => {
    expect(true).toBeTruthy();
  });
});
