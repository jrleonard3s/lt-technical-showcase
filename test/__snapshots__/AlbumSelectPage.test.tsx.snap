// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AlbumSelectPage > snapshot 1`] = `
{
  "foo": "bar",
}
`;
